### PR TITLE
PR: Add TE226 v2 colour checker reference values.

### PR DESCRIPTION
**Pull request by Rusching**
# Summary

Added TE226 V2 color checking values for issue: https://github.com/colour-science/colour/issues/901

# Preflight

**Code Style and Quality**

- [N/A] Unit tests have been implemented and passed.
- [N/A] Pyright static checking has been run and passed.
- [x] Pre-commit hooks have been run and passed.
- [N/A] New transformations have been added to the *Automatic Colour Conversion Graph*.
- [N/A] New transformations have been exported to the relevant namespaces, e.g. `colour`, `colour.models`.

**Documentation**

- [N/A] New features are documented along with examples if relevant.
- [N/A] The documentation is [Sphinx](https://www.sphinx-doc.org/en/master/) and [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant.

### CODE REVIEW

File: colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py [hunk 1]
```diff
@@ -386,13 +387,95 @@ class ColourChecker(
...
+    "patch27": np.array([0.1697, 0.1294, 0.7026]),
+    "patch28": np.array([0.2143, 0.1564, 0.1908]),
+    "patch29": np.array([0.1659, 0.3876, 0.3945]),
+    "patch30": np.array([0.1869, 0.1093, 0.7069]),
+    "patch31": np.array([0.3316, 0.1596, 0.1714]),
+    "patch32": np.array([0.8298, 0.8910, 0.5199]),
+    "patch33": np.array([0.1412, 0.1758, 0.4643]),
+    "patch34": np.array([0.0153, 0.0668, 0.0694]),
+    "patch35": np.array([0.6053, 0.5088, 0.1593]),
+    "patch36": np.array([0.4217, 0.4459, 0.3173]),
```
Comment (KelSolaar) [inline colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py:426]: **Minor**

Fine to keep the space between `patch` and the patch number. `patch19` --> `patch 19`.

File: colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py [hunk 2]
```diff
@@ -386,13 +387,95 @@ class ColourChecker(
...
+    "neutral 63": np.array([0.6050, 0.6401, 0.6958]),
+    "neutral 44": np.array([0.4119, 0.4358, 0.4724]),
+    "neutral 28": np.array([0.2638, 0.2798, 0.3018]),
+    "neutral 15": np.array([0.1405, 0.1489, 0.1598]),
+    "neutral 7": np.array([0.0628, 0.0665, 0.0701]),
+    "neutral 2": np.array(
+        [
+            0.0190,
+            0.0202,
+            0.0202,
```
Comment (KelSolaar) [inline colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py:438]: **Minor**

Extra coma after `0.0202` forces an inconsistent formatting wrap.

File: colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py [hunk 3]
```diff
@@ -386,13 +387,95 @@ class ColourChecker(
...
+    ),
+    "neutral < 0.1": np.array([0.0000, 0.0001, 0.0000]),
+}
+
+DATA_TE226_V2: dict = dict(
+    zip(
+        tuple(DATA_TE226_V2_CIE_XYZ.keys()),
+        XYZ_to_xyY(
+            list(DATA_TE226_V2_CIE_XYZ.values()),
+            CCS_ILLUMINANTS["CIE 1931 2 Degree Standard Observer"]["ICC D50"],
```
Comment (KelSolaar) [inline colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py:449]: **Major**

The whitepoint is actually D65:

image

image

**Review by KelSolaar — CHANGES_REQUESTED** [review]
Thanks @Rusching!

I left a few notes, beyond those, would it be possible to:

#### Update the header of the module:

```python
-   :attr:`colour.characterisation.datasets.colour_checkers.chromaticity_coordinates. CCS_TE226_V2`: Reference data from *TE226 V2*.
```

#### Add citation in alphabetical order:

```
-   :cite:`ImageEngineering2017` : Image Engineering. (2017). TE226 V2 data sheet. https://www.image-engineering.de/content/products/charts/te226/downloads/TE226_D_data_sheet.pdf
```

#### Use :cite:`ImageEngineering2017` wherever relevant in the module.

Cheers,

Thomas

File: colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py (hunk 3 above)
Comment (Rusching) [reply colour/characterisation/datasets/colour_checkers/chromaticity_coordinates.py]: Thank you for your detailed review and advice! I would push the changes.

**Review by KelSolaar — APPROVED** [review]

**Comment by KelSolaar** [comment]
Thanks @Rusching! It is merged! :)

**Comment by Rusching** [comment]
Happy! ^ ^

**Pull request merged**